packages/gestalt/src/Link.js

```diff
@@ -18,6 +18,59 @@ import getRoundingClassName from './getRoundingClassName.js';
 import focusStyles from './Focus.css';
 import useFocusVisible from './useFocusVisible.js';
 import layoutStyles from './Layout.css';
+import Icon from './Icon.js';
+import Box from './Box.js';
+import Text from './Text.js';
+
+type ExternalLinkIcon =
```

Comment:
I'm not a huge fan of the polymorphism here, but I also don't have any better ideas. 😬 This works, I think

Reply:
I thought this was neater than having 2 props: `isExternal` and `externalIcon`

Reply:
Agreed!